.github/workflows/asset_dispatch.yml

```diff
@@ -15,12 +15,12 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - name: Export node version
```

Comment:
'Export node version' step can be removed since we get the node version from the file now directly.